data/events/scripts/player.lua

```diff
@@ -273,3 +273,26 @@ function Player:onGainSkillTries(skill, tries)
 	end
 	return tries * configManager.getNumber(configKeys.RATE_SKILL)
 end
+
+function Player:onWrapItem(item)
+	local tile = Tile(item:getPosition())
+	if tile and not tile:getHouse() then
+		self:sendTextMessage(MESSAGE_STATUS_SMALL, "You can only wrap and unwrap items inside a house.")
```

Comment:
The official message is "You can only wrap and unwrap this item inside a house."